lib/context.js

```diff
@@ -527,7 +527,15 @@ api.createTermDefinition = ({
         {code: 'invalid type mapping', context: localCtx});
     }
 
-    if(type !== '@id' && type !== '@vocab' && type !== '@json') {
+    if((type === '@json' || type === '@none')) {
+      if(api.processingMode(activeCtx, 1.0)) {
+        throw new JsonLdError(
+          'Invalid JSON-LD syntax; an @context @type value not be ' +
+          '#{type} in JSON-LD 1.0 mode.',
```

Comment:
Need to use backticks and dollar sign to do interpolation.  Seems good to add quotes around the type too.
```suggestion
          `"${type}" in JSON-LD 1.0 mode.`,
```